### PR TITLE
[12.0][ADD] helpdesk_mgmt_timesheet_readonly

### PR DESCRIPTION
**Pull request by dalonsod**
This module adds a custom group with readonly permissions for combined tickets & timesheets within an independent category:

imagen

@lmiguens-solvos could you review code and test? Thanks!

cc @lromero-solvos @ChristianSantamaria

### CODE REVIEW

File: helpdesk_mgmt_timesheet_readonly/security/helpdesk_mgmt_timesheet_readonly.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <record id="group_helpdesk_timesheet_readonly" model="res.groups">
+            <field name="name">User: Readonly tickets &amp; timesheets</field>
+            <field name="category_id" ref="module_helpdesk_timesheet_category"/>
+        </record>
+
+    </data>
+
+    <data noupdate="1">
+
+        <record id="helpdesk_ticket_timesheet_user_rule" model="ir.rule">
```
Comment (lmiguens-solvos) [inline helpdesk_mgmt_timesheet_readonly/security/helpdesk_mgmt_timesheet_readonly.xml:14]: I assume that the perm_read will be by default True, but would it not be clearer if it was indicated?

File: helpdesk_mgmt_timesheet_readonly/security/helpdesk_mgmt_timesheet_readonly.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+
+        <record id="group_helpdesk_timesheet_readonly" model="res.groups">
```
Comment (lmiguens-solvos) [inline helpdesk_mgmt_timesheet_readonly/security/helpdesk_mgmt_timesheet_readonly.xml:5]: Doesn't have any implied_ids?

**Review by lmiguens-solvos — COMMENTED** [review]
I put a couple of comments in a file, but seeing everything together I understand it and, just looking at code, it seems correct. 
Good job!

File: helpdesk_mgmt_timesheet_readonly/security/helpdesk_mgmt_timesheet_readonly.xml (hunk 2 above)
Comment (lmiguens-solvos) [reply helpdesk_mgmt_timesheet_readonly/security/helpdesk_mgmt_timesheet_readonly.xml:5]: I see that it is to make only the Helpdesk and Timesheet menus visible later

File: helpdesk_mgmt_timesheet_readonly/security/helpdesk_mgmt_timesheet_readonly.xml (hunk 1 above)
Comment (dalonsod) [reply helpdesk_mgmt_timesheet_readonly/security/helpdesk_mgmt_timesheet_readonly.xml:14]: Conventions say that by default `perm_*` fields are `True` , thus there's no need to fill them. But, in this case I agree with you, the clearest way is filling all the fields. I'll change it

File: helpdesk_mgmt_timesheet_readonly/security/helpdesk_mgmt_timesheet_readonly.xml (hunk 2 above)
Comment (dalonsod) [reply helpdesk_mgmt_timesheet_readonly/security/helpdesk_mgmt_timesheet_readonly.xml:5]: You're right. 

The only problem is that for now is too restrictive (e.g. clicking on Many2one field link within a ticket, like team will fire a permissions error), but we can add readonly ACLs later

**Pull request merged**